Makefile

```diff
@@ -69,6 +69,7 @@ docker_wipe: docker_check warn_destructive prompt_user ## [WARNING] Remove all t
 
 .PHONY: localnet_up
 localnet_up: ## Starts localnet
+	make localnet_regenesis
```

Comment:
Each developer will now generate genesis themselves using the ignite's config.yml. It'll be more like protobufs - they are generated out of proto definitions.